src/Ubiquity/orm/creator/Model.php

```diff
@@ -119,9 +119,9 @@ public function getManyToOneMembers() {
 	private function getToStringField(){
 		$result=null;
 		foreach ( $this->members as $member ) {
-			if ($member->getDbType()!=="mixed" && $member->isNullable()!==true && !$member->isPrimary()) {
-				$result=$member->getName();
-			}
+            if ($member->isEmail()) {
```

Comment:
This verification is too restrictive.
And no value is returned if no email field is found....
It makes more sense to exclude some fields based on their names, rather than trying to search for an hypothetical email field

Reply:
Hi @jcheron ,

Well I understand what you mean. 

My idea was really a bit restrictive but I thought that it will be okay to return the ID of a model when the function returns null because your other function `getToString()` is programmed like this. 

I will do a rollback of all the files and I implement a solution in the function `getToStringField()`.
I have already an idea to do.

Best regards,
ndL-1990